tests/Orchestration/Adapter/DockerAPITest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Utopia\Tests\Adapter;
+
+use Utopia\Orchestration\Adapter\DockerAPI;
+use Utopia\Orchestration\Orchestration;
+use Utopia\Tests\Base;
+
+class DockerAPITest extends Base {
+    /**
+     * @var Orchestration
+     */
+
+    static $orchestration = null;
+
+    /**
+     * Return name of adapter
+     *
+     * @return string
+     */
+
+    static function getAdapterName(): string
+    {
+    return "Docker API";
```

Comment:
```suggestion
        return "Docker API";
```

Indentation is wrong